src/app/component/layout/components/header/header.component.scss

```diff
@@ -424,16 +424,15 @@ $screen-320: " screen and (max-width : 320px)";
   height: 36px;
 
   ul {
-    display: flex;
-    flex-direction: column;
+    display: inline;
     position: absolute;
     margin: 0;
     padding: 10px 0 0 0;
     background: #fff;
 
     li {
-      width: 100%;
-      height: 30px;
+      width: 141px;
```

Comment:
Why did you add the fixed width?

Reply:
I added width according to mockup.
![image](https://user-images.githubusercontent.com/50843868/97441353-4f551500-1931-11eb-8d24-181918510006.png)